controller/response/responder.go

```diff
@@ -151,8 +151,8 @@ func RespondWithApiError(w http.ResponseWriter, r *http.Request, requestId strin
 	data := &rest_model.APIErrorEnvelope{
 		Error: apierror.ToRestModel(apiError, requestId),
 		Meta: &rest_model.Meta{
```

Comment:
Should I fix the spelling of enrollment while I'm in here @andrewpmartinez ?

Reply:
Oh my. Yes - Nothing keys off it yet.